frames/src/main/java/com/checkout/frames/cvvinputfield/api/InternalCVVComponentMediator.kt

```diff
@@ -57,6 +57,14 @@ internal class InternalCVVComponentMediator(
         setContent { InternalCVVComponent() }
     }
 
+    override fun provideCvvComponentContent(
+        container: View,
+    ): View = ComposeView(container.context).apply {
+        // Dispose of the Composition when the view's LifecycleOwner is destroyed
+        setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+        setContent { InternalCVVComponent() }
+    }
```

Comment:
this can be this?
```suggestion
    override fun provideCvvComponentContent(
        container: View,
    ): View = provideCvvComponentContent(view,     ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
```